sys/kern/vm_map.c

```diff
@@ -161,33 +165,53 @@ void vm_map_entry_destroy(vm_map_t *map, vm_map_entry_t *ent) {
   vm_map_entry_free(ent);
 }
 
+static inline vm_map_entry_t *vm_map_entry_copy(vm_map_entry_t *src) {
+  if (src->object)
+    vm_object_hold(src->object);
```

Comment:
This case is not tested. `src->object` is always a pointer to something else than null.

Reply:
I will remove it and add when we will have this case in Mimiker.

Reply:
Yes, but I will leave it. We do exactly the same in `vm_map_entry_free`. I think it should be here, because if we will have that case we wouldn't have to look for code which assumes that `vm_object` is present.